src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -59,6 +376,40 @@ def __init__(self, sc, *args):
             self.ml = sc._jvm.org.apache.sysml.api.MLContext(sc._jsc, monitorPerformance, setForcedSparkExecType)
```

Comment:
@MechCoder Good suggestion. Let's improve the API of MLContext in a separate PR as part of https://issues.apache.org/jira/browse/SYSTEMML-234. A good task for someone to be a SystemML contributor :)